README.md

```diff
@@ -26,7 +26,7 @@
 
```

Comment:
- [OPTIONAL] It would be better to remove broken links like screenshot and Project Live Link in your README file until you get a good one. ♻️

Reply:
Okay, thank you!